pages/CheckingPage.js

```diff
@@ -0,0 +1,66 @@
+import BasePage from '../pages/BasePage';
+
+class CheckingPage extends BasePage {
+
+    get standardCheckingButton (){return $('label[for="Standard Checking"]')}
+
+    get interestCheckingButton (){return $('label[for="Interest Checking"]')}
+  
+    get individualCheckingButton (){return $('#Individual')}
+  
+    get jointCheckingButton (){return $('#Joint')}
+  
+    get checkingNameInput (){return $('#name')}
+  
+    get checkingDepositInput (){return $('#openingBalance')}
+  
+    get newCheckingFormSubmitButton () {return $('#newCheckingSubmit')}
+
+    get newCheckingFormResetButton () {return $('#newCheckingReset')}
+
+    get newCheckingConfirmation () {return $('#new-account-conf-alert')}
+
+    get newCheckingErrorText () {return $('#new-account-error-msg')}
+
+    get switchChecking () { return $('.switch-label')}
+
+    get checkingBody () { return $('.even')}
```

Comment:
This is not a robust selector. See if you can craft one that better individualizes this element. I assume this is the "card" that represents the account after account creation.

Reply:
I dont use it as a selector, as tr in the table are named by odd and even, when I create a Checking that card is selected so when I do the switch to the one Digital Bank provides me if a tr is even appears that means I change the checking view because the checking I created has only one transaction

Reply:
I understand. This is, IMO, too reliant on the specific data is present in the environment at the moment you ran the test. If you ever make a new transaction to the account you just created it'll be selected instead, breaking your test.

I think a more robust approach would be relying on the value of the switch after you activate it (it should only display one as "checked", the rest as "unchecked"). This approach won't be validating that the table actually displays different data, but at least it's not bounded to specific data being loaded on the environment for it to work properly.

Reply:
The values in the switch dont change when they are off or on